ovos_plugin_common_play/ocp/__init__.py

```diff
@@ -71,20 +70,33 @@ def __init__(self, bus=None, lang=None, settings=None):
             self.remove_event("mycroft.ready")
             self.replace_mycroft_cps(skills_ready)
         try:
+            # TODO: Should this just happen at install time? A user might not
+            #       want this shortcut.
```

Comment:
agree, this should be done in setup.py maybe?

Reply:
Yeah, I think the HomeAssistant plugin does it that way?

Reply:
Moved to separate issue #71